src/main/java/io/sphere/sdk/meta/TroubleshootingGuide.java

```diff
@@ -127,7 +127,11 @@
 
  <p>If you get do not get an exception, it may be of importance to know your JVM SDK version, Java version, date of occurrence and operating system.</p>
  Describe the behavior you observed and the one you would like to obtain.
-
+ <h3 id="product-not-found-by-text">Product not found</h3>
+ <ul>
+    <li>Maybe the product is not published. See {@link io.sphere.sdk.products.commands.updateactions.Publish}.</li>
+    <li>The product was not found for the search because the locale was not activated for search. Enable it in the Merchant Center.</li>
```

Comment:
What do you mean with "locale not being activated for search"? Sounds like for every language you can choose whether to make them "searchable" or not, but I guess that's not it.